src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateRuntimesRule.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.ProjectJsonMigration.Transforms;
+
+namespace Microsoft.DotNet.ProjectJsonMigration.Rules
+{
+    public class MigrateRuntimesRule : IMigrationRule
+    {
+        private readonly ITransformApplicator _transformApplicator;
+
+        public MigrateRuntimesRule(ITransformApplicator transformApplicator = null)
+        {
+            _transformApplicator = transformApplicator ?? new TransformApplicator();
+        }
+
+        public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migrationRuleInputs)
+        {
+            var propertyGroup = migrationRuleInputs.CommonPropertyGroup;
+
+            var propertyTransform = new AddPropertyTransform<IList<string>>(
```

Comment:
You can have this as a property of the class, like in the other Rules. Let's keep it consistent.